bcda/main_test.go

```diff
@@ -44,6 +44,7 @@ func (s *MainTestSuite) SetupSuite() {
 
 func (s *MainTestSuite) SetupTest() {
 	s.testApp = setUpApp()
+	models.InitializeGormModels()
```

Comment:
should this be autoMigrate()?  also, if we are doing this in the test setup should we remove it from the tests that are calling it now?  I see it in a few tests below like TestArchiveExpiringWithThreshold